collections-lists/FlatteningNestedCollectionInC#/FlatteningNestedCollectionInC#.Tests/UnitTests.cs

```diff
@@ -0,0 +1,140 @@
+namespace FlatteningNestedCollection.Tests;
+
+public class UnitTests
```

Comment:
Rename class name to more meaningful name.
Along with count, we should also check the data which is returned by each method.

Reply:
Renamed it. I added tests based on data for every case.